src/usr/local/www/system.php

```diff
@@ -391,7 +391,8 @@ function is_timezone($elt) {
 	))->setHelp(($i == 4) ? 'Address':null);
 
 	$help = "Enter IP addresses to be used by the system for DNS resolution. " .
-		"These are also used for the DHCP service, DNS forwarder and for PPTP VPN clients.";
+		"These are also used for the DHCP service, DNS Forwarder and DNS Resolver " .
+		"(if it has DNS Query Forwarding enabled).";
```

Comment:
This reads a bit odd. How about "... and DNS Resolver when it has DNS Query Forwarding enabled"